homeassistant/components/boschshc/manifest.json

```
@@ -0,0 +1,15 @@
+{
+  "domain": "boschshc",
```

We prefer to separate words with underscore.
```suggestion
  "domain": "bosch_shc",
```